commit 856ef55e7e1fb411cd42b917bac2a7aaf75344ae
Merge: 267214a2319b 30a8a105f0ce
Author: Dan Williams <dan.j.williams@intel.com>
Date:   Sun Apr 23 12:09:56 2023 -0700

    Merge branch 'for-6.4/cxl-poison' into for-6.4/cxl
    
    Include the poison list and injection infrastructure from Alison for
    v6.4.

diff --cc drivers/cxl/cxlmem.h
index 001dabf0231b,80276f37b78f..6090611669a6
--- a/drivers/cxl/cxlmem.h
+++ b/drivers/cxl/cxlmem.h
@@@ -249,7 -280,9 +280,8 @@@ struct cxl_poison_state 
   * @component_reg_phys: register base of component registers
   * @info: Cached DVSEC information about the device.
   * @serial: PCIe Device Serial Number
 - * @doe_mbs: PCI DOE mailbox array
   * @event: event log driver state
+  * @poison: poison driver state info
   * @mbox_send: @dev specific transport for transmitting mailbox commands
   *
   * See section 8.2.9.5.2 Capacity Configuration and Label Storage for
@@@ -286,7 -319,10 +318,8 @@@ struct cxl_dev_state 
  	resource_size_t component_reg_phys;
  	u64 serial;
  
 -	struct xarray doe_mbs;
 -
  	struct cxl_event_state event;
+ 	struct cxl_poison_state poison;
  
  	int (*mbox_send)(struct cxl_dev_state *cxlds, struct cxl_mbox_cmd *cmd);
  };